main.py

```
@@ -345,7 +403,20 @@ def _should_process(self, event: AstrMessageEvent) -> bool:
                 )
                 return False
 
-            blocked_by_provider_wake_prefix = self._is_blocked_by_provider_wake_prefix(event)
+            # 开启系统级唤醒词后：以 AstrBot wake_prefix 开头的消息等价点名唤醒
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)